docs/First-Installation.md

```diff
@@ -21,65 +21,58 @@ The simplest way to get most of these components is through an all-in-one kit su
-3. For Linux you can use the `dd` command, [Etcher](https://www.balena.io/etcher/) or [Raspberry Pi Imager](https://www.raspberrypi.org/software/).
-
-See the official Raspberry Pi "WRITING AN IMAGE TO THE SD CARD" [instructions](https://www.raspberrypi.org/documentation/installation/installing-images/).
-
-**Note** RetroPie is built on top of Raspberry Pi OS Buster (a Linux based OS for the Raspberry Pi) and as such the partition on the SD card is EXT4 (a linux filesystem). This partition is is not visible on Windows systems, so the card will show up as a smaller size than usual and you won't be able to see everything on the card, but it is all there. You will be able to access the filesystem over the network as described in the transferring roms section below.
-
-If you're updating from a previous version of RetroPie see [**HERE**](Updating-RetroPie).
+### Preferred Method
+
+1. First, insert the MicroSD card into into your computer.
+2. Download the official [Raspberry Pi Imager](https://www.raspberrypi.org/software/), which allows downloading the different OS versions of RetroPie.
+3. Click the **Choose OS** button
+4. Select **Emulation and game OS**
+5. Select **RetroPie**.
+6. Select the image for the model of Raspberry Pi (RPI) that you have. For example, if you have a Raspberry Pi 4, select the RPI 4/400 option ([example](https://user-images.githubusercontent.com/540857/107868009-493a6400-6e4e-11eb-9272-7f45d569dc44.gif)) 
+   * If you don't know which model Raspberry Pi you have, there are a few methods to find out: 
+     * The [Raspberry Pi Wikipedia page](https://en.wikipedia.org/wiki/Raspberry_Pi#Connectors) has user-friendly graphics to help determine model by looking at the board itself 
+     * If you have SSH/commandline access, run `cat /proc/device-tree/model` and it will output your Pi's model
+1. Click the **Choose SD Card** button. 
+2. The MicroSD card you plugged in earlier should appear in this list. Select it to continue
+3. Once your "OS" and "SD card" have been chosen, select **Write**, and wait until the operation completes
+4. Remove the MicroSD card from your PC, slide it back into the slot on your Raspberry Pi, and turn it on
+
+### Alternative Method
+
+1. [Download the RetroPie image](https://retropie.org.uk/download/) from the official website for your Raspberry Pi model.
+2. Download a program to write the image to your SD card.
+   * For Windows: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), or [Win32DiskImager](https://sourceforge.net/projects/win32diskimager/)
+   * For macOS: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), or [Apple Pi Baker](https://www.tweaking4all.com/hardware/raspberry-pi/applepi-baker-v2/)
```

Comment:
macOs can also use `dd`

Reply:
For some reason I can't comment on your above remark about listing a computer as a required item - Agreed.